help.go

```diff
@@ -174,24 +174,8 @@ func printNodeDetail(w *helpWriter, node *Node, hide bool) {
 		w.Print("Arguments:")
 		writePositionals(w.Indent(), node.Positional)
```

Comment:
writePositionals -> PrintPositionals?

Reply:
I went in the opposite direction since it made more sense to me. Print -> Write for the new public funcs since it writes to WriteHelper but i can switch to print if you want.